src/custom/hooks/useContract/index.ts

```diff
@@ -42,6 +42,7 @@ export function useENSRegistrarContract(withSignerIfPossible?: boolean): Contrac
     switch (chainId) {
       case ChainId.MAINNET:
       case ChainId.RINKEBY:
+      case ChainId.GOERLI: // TODO: check this
```

Comment:
Looks like it's the same https://goerli.etherscan.io/address/0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e#code=